pkg/executables/clusterctl_test.go

```diff
@@ -95,7 +95,7 @@ func TestClusterctlInitInfrastructure(t *testing.T) {
 			wantExecArgs: []interface{}{
 				"init", "--core", core, "--bootstrap", bootstrap, "--control-plane", controlPlane, "--infrastructure", "aws:v0.6.4", "--config", test.OfType("string"),
 				"--bootstrap", etcdadmBootstrap, "--bootstrap", etcdadmController,
-				"--watching-namespace", constants.EksaSystemNamespace,
```

Comment:
I think we want the watching-namespace to be there for v1alpha1 (for types other than DockerDatacenterKind

Reply:
For reference: https://github.com/aws/eks-anywhere/issues/810